libdnf5-plugins/python_plugins_loader/python_plugins_loader.cpp

```diff
@@ -232,37 +245,51 @@ void PythonPluginLoader::load_plugin_file(const fs::path & file_path) {
         fetch_python_error_to_exception("PyDict_CallMethod(base_class, \"get_locked_base\", NULL): ");
     }
     UniquePtrPyObject add_plugin_string(PyUnicode_FromString("add_plugin"));
-    PyObject_CallMethodObjArgs(locked_base.get(), add_plugin_string.get(), plugin_instance, NULL);
+    UniquePtrPyObject plugin_name_string(PyUnicode_FromString(plugin_name.c_str()));
+    type = SWIG_TypeQuery("libdnf5::ConfigParser *");
+    PyObject * py_config_parser = SWIG_NewPointerObj(plugin_config, type, 0);
```

Comment:
Update the code to release the `plugin_config` Python wrapper after use and to work with the revised API using `&&` (move semantics).
```
    UniquePtrPyObject config_parser(
        SWIG_NewPointerObj(new libdnf5::ConfigParser(std::move(*plugin_config)), type, SWIG_POINTER_OWN));
    if (!config_parser) {
        fetch_python_error_to_exception("SWIG_NewPointerObj(plugin_config, type, SWIG_POINTER_OWN)");
    }
    PyObject_CallMethodObjArgs(
        locked_base.get(),
        add_plugin_string.get(),
        plugin_name_string.get(),
        config_parser.get(),
        plugin_instance,
        NULL);
```